src/ngraph/frontend/onnx_import/exceptions.hpp

```diff
@@ -24,12 +25,10 @@ namespace ngraph
     {
         namespace error
         {
-            struct NotSupported : ngraph_error
+            struct NotSupported : AssertionFailure
             {
-                explicit NotSupported(const std::string& op_name,
-                                      const std::string& name,
-                                      const std::string& message)
-                    : ngraph_error{op_name + " node (" + name + "): " + message}
+                explicit NotSupported(const std::string& what_arg)
```

Comment:
What is `what_arg` ? What information should I pass here?

Reply:
Seems to be the standard name for this thing.

> `what_arg` - explanatory string that can be accessed through what().

https://en.cppreference.com/w/cpp/error/runtime_error